leetcode/src/2222.c

```diff
@@ -0,0 +1,27 @@
+long long numberOfWaysForChar(char * s, char c){
```

Comment:
Non blocking comment, sometimes `leetcode` names things that make sense in the context of the problem but not generally in the context of code (just my opinion 😂) therefore when working on code that would be read by more people, it is usually better to name things appropriately to help people reading your code.

Things like `firstItemsNumber` and `secondItemsNumber` doesn't explain anything when reading this code and have to be "deciphered" by the reader. So naming things appropriately is very much appreciated but again this is not a blocking comment.

Reply:
fixed, please review.